### PR TITLE
Add forceResizeWidth on GUI Text Block to allow for width resize even…

### PR DESCRIPTION
**Pull request by carolhmj**
… if textWrap is not clip.

Related forum question: https://forum.babylonjs.com/t/gui-alignment-for-chat-bubbles/37573

Related PG link: https://playground.babylonjs.com/#XCPP9Y#17254

### CODE REVIEW

**Comment by bjsplat** [comment]
Snapshot stored with reference name:
refs/pull/13460/merge

Test environment: 
https://babylonsnapshots.z22.web.core.windows.net/refs/pull/13460/merge/index.html

To test a playground add it to the URL, for example:

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/13460/merge/index.html#WGZLGJ#4600

Links to test babylon tools with this snapshot:

https://playground.babylonjs.com/?snapshot=refs/pull/13460/merge
https://sandbox.babylonjs.com/?snapshot=refs/pull/13460/merge
https://gui.babylonjs.com/?snapshot=refs/pull/13460/merge
https://nme.babylonjs.com/?snapshot=refs/pull/13460/merge

To test the snapshot in the playground with a playground ID add it after the snapshot query string:

https://playground.babylonjs.com/?snapshot=refs/pull/13460/merge#BCU1XR#0

**Comment by bjsplat** [comment]
Visualization tests for webgl1 have failed. If some tests failed because the snapshots do not match, the report can be found at

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/13460/merge/testResults/webgl1/index.html

If tests were successful afterwards, this report might not be available anymore.

**Comment by bjsplat** [comment]
Visualization tests for webgl2 have failed. If some tests failed because the snapshots do not match, the report can be found at

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/13460/merge/testResults/webgl2/index.html

If tests were successful afterwards, this report might not be available anymore.

**Review by Popov72 — APPROVED** [review]

**Review by sebavan — APPROVED** [review]

**Comment by sebavan** [comment]
@carolhmj there is smthg not working in the way you generate the GUI test :-( for weird reasons let s check with @RaananW as it is also blocking https://github.com/BabylonJS/Babylon.js/pull/13361

**Review by Popov72 — APPROVED** [review]

**Comment by bjsplat** [comment]
Visualization tests for webgl1 have failed. If some tests failed because the snapshots do not match, the report can be found at

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/13460/merge/testResults/webgl1/index.html

If tests were successful afterwards, this report might not be available anymore.

**Comment by bjsplat** [comment]
Visualization tests for webgl2 have failed. If some tests failed because the snapshots do not match, the report can be found at

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/13460/merge/testResults/webgl2/index.html

If tests were successful afterwards, this report might not be available anymore.

**Comment by RaananW** [comment]
We'll check that together tomorrow

**Comment by bjsplat** [comment]
Visualization tests for webgl2 have failed. If some tests failed because the snapshots do not match, the report can be found at

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/13460/merge/testResults/webgl2/index.html

If tests were successful afterwards, this report might not be available anymore.

**Comment by bjsplat** [comment]
Visualization tests for webgl1 have failed. If some tests failed because the snapshots do not match, the report can be found at

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/13460/merge/testResults/webgl1/index.html

If tests were successful afterwards, this report might not be available anymore.

**Comment by bjsplat** [comment]
Visualization tests for webgl1 have failed. If some tests failed because the snapshots do not match, the report can be found at

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/13460/merge/testResults/webgl1/index.html

If tests were successful afterwards, this report might not be available anymore.

**Comment by bjsplat** [comment]
Visualization tests for webgl2 have failed. If some tests failed because the snapshots do not match, the report can be found at

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/13460/merge/testResults/webgl2/index.html

If tests were successful afterwards, this report might not be available anymore.

File: packages/tools/tests/test/visualization/config.json [hunk 1]
```diff
@@ -1530,6 +1530,9 @@
             "renderCount": 10
         },
         {
+            "title": "GUI Force Resize Width",
+            "playgroundId": "#XCPP9Y#17259"
+        },
             "title": "GUI Gradient Linear",
             "playgroundId": "#XCPP9Y#17227"
         },
```
Comment (RaananW) [inline packages/tools/tests/test/visualization/config.json:1538]:
```suggestion
        }
```

File: packages/tools/tests/test/visualization/config.json [hunk 2]
```diff
@@ -1530,9 +1530,12 @@
             "renderCount": 10
         },
         {
+            "title": "GUI Force Resize Width",
+            "playgroundId": "#XCPP9Y#17259"
+        },
             "title": "GUI Gradient Linear",
             "playgroundId": "#XCPP9Y#17227"
-        },
+        }
```
Comment (RaananW) [inline packages/tools/tests/test/visualization/config.json:1538]:
```suggestion
        },
```

File: packages/tools/tests/test/visualization/config.json [hunk 3]
```diff
@@ -1530,6 +1530,9 @@
             "renderCount": 10
         },
         {
+            "title": "GUI Force Resize Width",
+            "playgroundId": "#XCPP9Y#17259"
+        },
             "title": "GUI Gradient Linear",
```
Comment (RaananW) [inline packages/tools/tests/test/visualization/config.json:1536]:
```suggestion
        {
            "title": "GUI Gradient Linear",
```

**Pull request merged**